jquery.flot.symbol.js

```diff
@@ -46,6 +47,7 @@ The symbols are accessed as strings through the standard symbol options:
             if (!shadow) {
                 ctx.lineTo(x, y - height / 2);
                 ctx.lineTo(x - size / 2, y + height / 2);
+                ctx.lineTo(x + size / 2, y + height / 2);
```

Comment:
The previous implementation used to closePath() after each shape which was making the left corners of the diamond and triangle to be nicely draw. Before adding this extra line, the issue was visible when the lineWidth > 1